src/styles/_catalog.scss

```diff
@@ -1,9 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  margin: auto;
+  justify-content: space-evenly;
```

Comment:
I think it's better to use justify-content: center and delete margin

Check the vertical gaps, because the justify-content: space-evenly has made them larger than they need to be